programmers/문자열-내-p와-y의-개수.js

```diff
@@ -0,0 +1,9 @@
+function solution(s) {
+  let p = 0, y = 0;
```

Comment:
준상님 코드도 좋지만 개인적으로는 아래 코드가 더 깔끔해 보이네요 :D
요것은 개취입니다^^
```suggestion
  let [p, y] = [0, 0];
```